README.md

```diff
@@ -13,23 +37,20 @@ name=venafi
 connector.class=com.opencredo.connect.venafi.tpp.log.TppLogSourceConnector
 tasks.max=1
 type.name=kafka-connect
-venafi.topic=TPP-LOGS
-venafi.base.url=https://localhost:443/vedsdk
-venafi.username=placeholder_username
-venafi.password=placeholder_password
-venafi.batch.size=100 
-venafi.poll.interval=1000
+venafi.base.url=<your_base_url>
+venafi.username=<your_api_username>
+venafi.password=<your_api_password>
 ```
-This is filled with the default values as provided by the [config definition class](./src/main/java/com/opencredo/connect/venafi/tpp/log/TppLogSourceConfig.java).  
-Whilst all `venafi` fields are currently optional and will default to above please change `venafi.base.url`, `venafi.username` and `venafi.password`.   
-4. Create a directory to place this files e.g. `<path-to-confluent>/share/kafka/plugins`.  
-5. Add this to the plugin path in your Connect properties file.   
-6. Then call: `bin/confluent load venafi -d venafi.properties`  
+This is filled with the minimum values required, any default values are provided by the [config definition class](./src/main/java/com/opencredo/connect/venafi/tpp/log/TppLogSourceConfig.java). 
+This can also be looked at for more information on configuration or look at the [wiki on the config definitions.](https://github.com/opencredo/kafka-connect-venafi-tpp/wiki/Config-Definitions-explained.)
+Make sure to replace the items with the <> brackets with what's needed to connect to your Venafi TPP/VEDSDK instance.   
+4. Create a directory to place the JAR file in and place it there e.g. `<path-to-confluent>/share/java/kafka-connect-venafi`.  
+5. Then call: `<path-to-confluent>/bin/confluent load venafi -d venafi.properties`  
 
 For more information please look at the [Confluent instructions on manually installing connectors](https://docs.confluent.io/current/connect/managing/install.html#connect-install-connectors).
 
 ---
-If you need to unload/reload it use: `bin/confluent unload venafi`  
+If you need to unload/reload it use: `<path-to-confluent>/bin/confluent unload venafi`  
```

Comment:
```suggestion
If you need to unload/reload the connector use: `<path-to-confluent>/bin/confluent unload venafi`  
```
comma before "use"? I think I'm currently mixing German and English rules, and it's not helping... ;-)

Reply:
I mean I think you're right.